modules/internal/ChapelSyncvar.chpl

```diff
@@ -137,13 +137,13 @@ module ChapelSyncvar {
     // ``a`` needs to be a ``valType``, not a sync.
     //
     pragma "dont disable remote value forwarding"
-    proc init(const other : _syncvar) {
+    proc init(const ref other : _syncvar) {
```

Comment:
They were a workaround for issues partway through this PR that I believe I have since resolved, but it seemed like an improvement to me. (It makes the standard module more understandable to us by being more explicit; and it reduces the chance of a bug impacting this standard module - expecting instead to find tests that fail related to `const` intent, say).

Reply:
I agree that this is a better way to write the formals.